src/app/app.module.ts

```diff
@@ -1,32 +1,43 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
 
-import { CartService } from './services/cart-service/cart.service';
-import { ProductService } from './services/product-service/product.service';
-
 import { AppComponent } from './app.component';
+import { DemoComponent } from './components/demo/demo.component';
 import { FooterComponent } from './components/footer/footer.component';
 import { HeaderComponent } from './components/header/header.component';
+import { ProductsPageComponent } from './modules/products/components/products-page/products-page.component';
 
 import { ProductsModule } from './modules/products/products.module';
+import { SharedModule } from './modules/shared/shared.module';
+import { CoreModule } from './modules/core/core.module';
+
+const routes: Routes = [
```

Comment:
better to extract routes into separate file

Reply:
These routes are temporary. I created it just to make it easier to work with the application in the browser. I have already moved all routes into routing module. Or is it better to move all routes constants into separate files?